src/test/msgr/test_async_networkstack.cc

```diff
@@ -20,6 +20,7 @@
 #include <string>
 #include <set>
 #include <vector>
+#include <random>
```

Comment:
The only nit-pick would be to put the header in alphabetical order, otherwise lgtm